ChromeExtension/js/apiKeyViewModel.js

```diff
@@ -1,75 +1,64 @@
-function apiKeyViewModel() {
-
-
-    var self = this;
-    self.apiKey = ko.observable('');
-    self.apiSecret = ko.observable('');
-    self.showBackLink = ko.observable(false);
-    self.showHelpLink = ko.observable(true);
-    self.newInstall = ko.observable(true);
-    self.UrlApiKeys = ko.observable('');
-
-
-    self.apiKey.subscribe(function (newValue) {
-        localStorage.setItem("apiKey", newValue);
-    });
-
-    self.apiSecret.subscribe(function (newValue) {
-        localStorage.setItem("apiSecret", newValue);
-    });
-
-
-    self.loadKey = function () {
-        var ak = localStorage["apiKey"];
-        if (typeof ak !== 'undefined') {
-            self.apiKey(ak);
-        }
-        var asa = localStorage["apiSecret"];
-        if (typeof asa !== 'undefined') {
-            self.apiSecret(asa);
-        }
+var apiKeyViewModel = {
```

Comment:
you'll see changes to all of the Knockout viewmodels due to this: https://developer.chrome.com/docs/extensions/develop/migrate/improve-security#remove-execution-of-strings

Manifest v3 has a new rule for Content Security Policy disables eval and new Function, and Knockout's default binder uses new Function to parse bindings.

So I need to make changes to all the viewmodels to use custom binding: https://github.com/brianmhunt/knockout-secure-binding